docs/source/examples/Notebook/rstversions/Typesetting Equations.rst

```diff
@@ -50,7 +50,7 @@ Source
     \end{equation*}
 
 Display
--------
+~~~~~~~
 
```

Comment:
@takluyver @minrk OK. It looks like nbconvert has a postprocesser option which would likely be the cleanest place to do the postprocessing and then update the makefiles for the notebook docs to use the postprocessor. Busy today but could take a look at next week or if someone wants to do a quicker fix, please do. Other than this which is not a new issue, this PR is good to merge.